src/ert/scheduler/scheduler.py

```diff
@@ -285,11 +287,11 @@ async def execute(
             await self.driver.finish()
             for scheduling_task in scheduling_tasks:
                 scheduling_task.cancel()
-            with suppress(asyncio.CancelledError):
-                await asyncio.wait(
-                    scheduling_tasks,
-                    return_when=asyncio.ALL_COMPLETED,
-                )
+            # we don't care about exceptions when all done
```

Comment:
"don't care" as in not log or print them?

Reply:
I should perhaps fix the comment. 
`We don't care about exceptions when cancelling the scheduling tasks`.
Or shall we collect the exceptions and log them? Hm, not sure if this makes sense since when cancelling does not go through it might just hang.